src/store/modules/ui.store.js

```diff
@@ -200,6 +207,21 @@ export default {
                 state.hostname.includes(hostname)
             )
         },
+        floatingPanelFeatureInfo(state, getters) {
```

Comment:
Keep the same nomenclature everywhere `floatingPanel` => `tooltip` => `tooltipFeatureInfo`